# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/navigation-timing/secure_connection_start_non_zero.https.html.ini
# Commit: 85883a720dab
# Full Hash: 85883a720dab61e82bcdb7391236dc06427466ee
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2021-04-07 15:47:02
# Regressor Bug: 1702830
# File Overlap Count: 1
# Description:
#   Bug 1702830 - [wpt-sync] Update web-platform-tests to 336ec39f06d15863a4af6b2aa6592530b37a5aee, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 336ec39f06d15863a4af6b2aa6592530b37a5aee
# ==============================================================================

diff -r acef12d74007 -r 85883a720dab testing/web-platform/meta/navigation-timing/secure_connection_start_non_zero.https.html.ini
--- a/testing/web-platform/meta/navigation-timing/secure_connection_start_non_zero.https.html.ini	Fri Apr 02 22:03:46 2021 +0000
+++ b/testing/web-platform/meta/navigation-timing/secure_connection_start_non_zero.https.html.ini	Fri Apr 02 22:06:52 2021 +0000
@@ -2,8 +2,7 @@
   [Test that secureConnectionStart is not zero]
     expected:
       if (os == "win") and ccov: [FAIL, PASS]
-      if (os == "linux") and webrender and fission and debug: ["FAIL", "PASS"]
-      if (os == "linux") and not webrender and not debug and (processor == "x86_64"): ["FAIL", "PASS"]
+      if (os == "linux") and not debug and (processor == "x86_64") and webrender and not fission: ["FAIL", "PASS"]
       if (os == "android") and debug and not webrender: ["FAIL", "PASS"]
       if (os == "win") and webrender: PASS
       if os == "mac": PASS